go.mod

```diff
@@ -4,26 +4,25 @@ go 1.16
 
 require (
 	github.com/Microsoft/go-winio v0.5.1
-	github.com/compose-spec/compose-go v1.0.4
+	github.com/compose-spec/compose-go v1.0.5
 	github.com/containerd/cgroups v1.0.2
 	github.com/containerd/console v1.0.3
-	github.com/containerd/containerd v1.6.0-beta.2.0.20211112054404-aa1b0736165c
-	github.com/containerd/containerd/api v1.6.0-beta.2.0.20211112054404-aa1b0736165c
+	github.com/containerd/containerd v1.6.0-beta.2.0.20211119054103-db9c6fc72276
```

Comment:
Waiting for beta.3 to be tagged https://github.com/containerd/containerd/pull/6267